artifactory/utils/golang/project/project.go

```diff
@@ -226,6 +289,19 @@ func (project *goProject) archiveProject(version, tempDir string) (string, error
 	return tempFile.Name(), nil
 }
 
+// Add the info file also as an artifact to be part of the build info.
+func (project *goProject) addInfoFileToBuildInfo(infoFilePath string) error {
+	fileDetails, err := fileutils.GetFileDetails(infoFilePath)
+	if err != nil {
+		return err
+	}
+
+	artifact := buildinfo.Artifact{Name: project.version + ".info"}
+	artifact.Checksum = &buildinfo.Checksum{Sha1: fileDetails.Checksum.Sha1, Md5: fileDetails.Checksum.Md5}
```

Comment:
Question - Haven't we already calculated checksum for the info file when we published it to Artifactory?
If so, we should reuse the calculated checksum and not calculate it again.

Reply:
No. The publish to Artifactory is using api/go and therefore we don't calculate the checksum. The checksum is being calculated once at this stage. We can't use checksum deploy since the url is build with api/go